docs/source/getting-started/assessing-utility/noise.ipynb

```diff
@@ -0,0 +1,527 @@
+    "                 group_by(\"ILOSTAT\").\n",
+    "                 agg(pl.len().alias(\"Actual\"))\n",
+    "                ).collect()\n",
+    "\n",
+    "actual_values = actual_values.with_columns(\n",
+    "    pl.col(\"ILOSTAT\").map_elements(lambda x: labor_status_codes.get(x, x))\n",
+    ")\n",
+    "\n",
+    "actual_values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Remember that using Laplacian noise takes in one parameter, `epsilon`, so when using Laplacian noise, we don't need to define the context any differently than before."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (4, 2)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>ILOSTAT</th><th>Laplace Noise</th></tr><tr><td>str</td><td>u32</td></tr></thead><tbody><tr><td>&quot;Laid Off&quot;</td><td>76064</td></tr><tr><td>&quot;NA&quot;</td><td>35799</td></tr><tr><td>&quot;Working for Pa…</td><td>79765</td></tr><tr><td>&quot;Not Working Bu…</td><td>8118</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (4, 2)\n",
+       "┌──────────────────────────┬───────────────┐\n",
+       "│ ILOSTAT                  ┆ Laplace Noise │\n",
+       "│ ---                      ┆ ---           │\n",
+       "│ str                      ┆ u32           │\n",
+       "╞══════════════════════════╪═══════════════╡\n",
+       "│ Laid Off                 ┆ 76064         │\n",
+       "│ NA                       ┆ 35799         │\n",
+       "│ Working for Pay          ┆ 79765         │\n",
+       "│ Not Working But Employed ┆ 8118          │\n",
+       "└──────────────────────────┴───────────────┘"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "laplace_values = (context.query().\n",
+    "                  group_by(\"ILOSTAT\").\n",
+    "                  agg(\n",
+    "                      pl.len().\n",
+    "                      dp.laplace().\n",
+    "                      alias(\"Laplace Noise\")\n",
+    "                      )\n",
+    "                ).release().collect()\n",
+    "\n",
+    "laplace_values = laplace_values.with_columns(\n",
+    "    pl.col(\"ILOSTAT\").map_elements(lambda x: labor_status_codes.get(x, x))\n",
+    ")\n",
+    "\n",
+    "laplace_values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use the Gaussian mechanism for adding noise, we specify a different parameter, `rho`, instead of `epsilon`; `rho` is set to 1 for this tutorial. See `TODO` for a fuller discussion of setting a privacy budget. "
```

Comment:
One issue here is that we're talking about ρ-zCDP in this line without giving any context on that. Previously, we only talk about approx DP, but here in the tutorial, we don't really do that, but use zCDP, instead.

Reply:
I took out this line from the introduction: Otherwise, the Gaussian distribution may be used to satisfy zero concentrated DP...

is that what you meant?